src/style.css

```diff
@@ -1 +1,61 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav {
+  padding: 11px 0;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  padding-left: 0;
+}
+
+.nav__list-item + .nav__list-item {
+  margin-left: 20px;
+}
+
+.nav__list-link {
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #000;
+  padding: 23px 0;
+  transition: all ease 0.3s;
+}
+
+.is-active {
+  color: #00acdc;
+  position: relative;
+}
+
+.is-active::before {
+  content: "";
+  position: absolute;
+  height: 4px;
+  background-color: #00acdc;
+  bottom: 0;
+  right: 0;
+  left: 0;
+  border-radius: 8px;
+}
+
+.nav__list-link:hover {
+  color: #00acdc;
+}
```

Comment:
Would be nice to group your styles, if you have styles for hover it should be near to element class, that will be more readable